File: chihun-jang/시저암호.js
function solution(s, n) {
  var mydict =
    "ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHIJKLMNOPQRSTUVWXYabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxy";
  var answer = "";

  for (var i = 0; i < s.length; i++) {
    if (s[i] === " ") {
      answer += " ";
    } else {
      answer += mydict[mydict.indexOf(s[i]) + n];
    }
  }

  return answer;
}

File: chihun-jang/시저암호.java
class Solution {
  public String solution(String s, int n) {
      String answer = "";
      int num = 0;
      for (int i = 0 ; i < s.length();i++){
          num = s.charAt(i);
          
          if (num == 32){
              num = num -n;
          }
          else if (num < 91){
              if(num+n >= 91){
                  num = num - 26;
              }
          }
          else{
              if(num+n>=123){
                  num = num-26;
              }
          }
          answer += Character.toString ((char) num+n);
      }
      return answer;
  }
}